meetings.md

```diff
@@ -20,7 +20,8 @@ title: Meetings
 {% assign ys = "" | split: ',' %}
 
 {% for path in paths %}
-    {% capture p %}{{ path | remove: "/assets/meetings/" | slice: 0, 7 }}{% endcapture %}
+<!-- {{path | remove: "/assets/meetings/"}} -->
```

Comment:
if this code isn't being actively used it should be removed, not just commented out

Reply:
Oopsy pretty sure i gaslit myself into thinking you put it there so left it, have removed now :)